styles/drawer.css

```diff
@@ -0,0 +1,5 @@
+
+.drawer-transition {
+    transition: 1s;
```

Comment:
Not sure if this was meant to stay as 1s but I like it either way

Reply:
I don't mind the 1s, if we need to change it, we can always come back to it.